packages/build-info/src/settings/get-build-settings.ts

```diff
@@ -79,7 +77,7 @@ async function getSettings(framework: Framework, project: Project, baseDirectory
     buildCommand: buildCommands[0],
     devCommand: devCommands[0],
     frameworkPort: framework.dev?.port,
-    dist: project.fs.join(baseDirectory, frameworkDist),
+    dist: project.fs.join(baseDirectory, framework.build.directory),
```

Comment:
Can we guarantee that this is always defined? Presumably we do set this for every framework

Reply:
yep it's required: 
![CleanShot 2023-07-05 at 11 22 19](https://github.com/netlify/build/assets/11156362/f486fd72-a453-4442-9a5a-11ab41cad23e)